src/plugins/ovp/components/medialive-channel-cards/selector.js

```diff
@@ -0,0 +1,13 @@
+import getMediaLive from '@triniti/cms/plugins/ovp/selectors/getMediaLive';
+import isGranted from '@triniti/cms/plugins/iam/selectors/isGranted';
+
+/**
+ * @param {Object} state
+ * @param {Object} ownProps
+ *
+ * @returns {Object}
+ */
+export default (state) => ({
+  getMediaLive: (nodeRef) => getMediaLive(state, nodeRef),
+  isPermissionGranted: isGranted(state, 'triniti:ovp.medialive:command:*'),
```

Comment:
@efjacobson might want to do `canStartChannel` or `canStopChannel`.  unless you feel confident that a user who can do this can do both.